packages/components/src/utils/tmp-ariakit-test-render-replacement.ts

```diff
@@ -0,0 +1,173 @@
+// See https://github.com/WordPress/gutenberg/pull/62947
+// and https://github.com/WordPress/gutenberg/pull/64066
```

Comment:
I was going to suggest putting a bit more descriptive comment for this file, but is seems like we can remove this fork very soon! 👏  https://github.com/ariakit/ariakit/pull/3978

Reply:
it's gone now!